### PR TITLE
GSOC'22- Root Project- Final Evaluation Blog for Inference Project

### PR DESCRIPTION
**Pull request by Neel-Shah-29**
- The Blog has all the details of my work done during GSOC 2022 at CERN HSF(Root Project). 
- My project was ROOT - TMVA SOFIE Developments - Inference Code Generation for Deep Learning models.
- The deadline for the final evaluation is 10th October,2022.

### CODE REVIEW

**Comment by Neel-Shah-29** [comment]
Hello @lmoneta, Kindly review the  blog for the final evaluation. Please let me know if any improvements are required. Thanks!

**Comment by agheata** [comment]
@lmoneta we need your approval before merging

**Review by lmoneta — APPROVED** [review]
Nice blog !
Thank you @Neel-Shah-29 !

File: _gsocblogs/2022/blog_ROOT_Neelshah.md [hunk 1]
```diff
@@ -158,13 +162,97 @@ CASE-2 \: We look then to the shape values, and if they are equal result is same
...
+np.concat() part was already implemented previously in the concat operator , i need to extend the support so it could support np.stack() functionality as well. This was required for GNN testing.
+
+- **PR Status**\:
+
+| Pull Request| PR Number |   Status     |
+| :---        |    :----:   |          ---: |
+| Extend Concat ONNX Operator to implement stack functionality as the ONNX ConcatFromSequence operator | [#11317](https://github.com/root-project/root/pull/11317)       | <img src="https://img.shields.io/badge/PR-Merged-blueviolet?style=for-the-badge&logo=appveyor"> |
+
+## Some Useful Blogs and Important Links
 1) [Python Tutorials for various C files of Tutorials/TMVA](https://gist.github.com/Neel-Shah-29/7e46bee55f7c09a18e94696a0a3e5ccf)
```
Comment (agheata) [inline _gsocblogs/2022/blog_ROOT_Neelshah.md:248]:
```suggestion
1) [Python Tutorials for various C files of Tutorials/TMVA](https://gist.github.com/Neel-Shah-29/7e46bee55f7c09a18e94696a0a3e5ccf)

```

File: _gsocblogs/2022/blog_ROOT_Neelshah.md [hunk 2]
```diff
@@ -158,13 +162,97 @@ CASE-2 \: We look then to the shape values, and if they are equal result is same
...
 2) [Documentation on RModelParser_ONNX.cxx](https://gist.github.com/Neel-Shah-29/5c1399ccd23903928128822c6f3e0957)
```
Comment (agheata) [inline _gsocblogs/2022/blog_ROOT_Neelshah.md:249]:
```suggestion
2) [Documentation on RModelParser_ONNX.cxx](https://gist.github.com/Neel-Shah-29/5c1399ccd23903928128822c6f3e0957)

```

File: _gsocblogs/2022/blog_ROOT_Neelshah.md [hunk 3]
```diff
@@ -158,13 +162,97 @@ CASE-2 \: We look then to the shape values, and if they are equal result is same
...
 3) [Getting into GSOC 2022](https://gist.github.com/Neel-Shah-29/3f04f05a8a353605068e32e55a5093c1)
```
Comment (agheata) [inline _gsocblogs/2022/blog_ROOT_Neelshah.md:250]:
```suggestion
3) [Getting into GSOC 2022](https://gist.github.com/Neel-Shah-29/3f04f05a8a353605068e32e55a5093c1)

```

File: _gsocblogs/2022/blog_ROOT_Neelshah.md [hunk 4]
```diff
@@ -158,13 +162,97 @@ CASE-2 \: We look then to the shape values, and if they are equal result is same
...
 4) [All about Community Bonding Period](https://gist.github.com/Neel-Shah-29/b2c887adaa118ba68b42e324d3b2a47a)
```
Comment (agheata) [inline _gsocblogs/2022/blog_ROOT_Neelshah.md:251]:
```suggestion
4) [All about Community Bonding Period](https://gist.github.com/Neel-Shah-29/b2c887adaa118ba68b42e324d3b2a47a)

```

File: _gsocblogs/2022/blog_ROOT_Neelshah.md [hunk 5]
```diff
@@ -158,13 +162,97 @@ CASE-2 \: We look then to the shape values, and if they are equal result is same
...
 5) [Implementing the Operators in Sofie](https://gist.github.com/Neel-Shah-29/f0371566ca1e24a6b3a9b4097cdd44db)
```
Comment (agheata) [inline _gsocblogs/2022/blog_ROOT_Neelshah.md:252]:
```suggestion
5) [Implementing the Operators in Sofie](https://gist.github.com/Neel-Shah-29/f0371566ca1e24a6b3a9b4097cdd44db)

```

File: _gsocblogs/2022/blog_ROOT_Neelshah.md [hunk 6]
```diff
@@ -158,13 +162,97 @@ CASE-2 \: We look then to the shape values, and if they are equal result is same
...
-6) [Mid Evaluation Detailed Report](https://gist.github.com/Neel-Shah-29/d51a9038dd07ef096127a62a92113fa0)
+6) [Final Evaluation Detailed Report](https://gist.github.com/Neel-Shah-29/d51a9038dd07ef096127a62a92113fa0)
```
Comment (agheata) [inline _gsocblogs/2022/blog_ROOT_Neelshah.md:253]:
```suggestion
6) [Final Evaluation Detailed Report](https://gist.github.com/Neel-Shah-29/d51a9038dd07ef096127a62a92113fa0)

```

File: _gsocblogs/2022/blog_ROOT_Neelshah.md [hunk 7]
```diff
@@ -158,13 +162,97 @@ CASE-2 \: We look then to the shape values, and if they are equal result is same
...
+7) [Final Project Presentation](https://docs.google.com/presentation/d/1VdOEkNsFcaBAo15kjVDRtJ_BWvVh9E6KgKzzrmkr1NQ/edit?usp=sharing)
```
Comment (agheata) [inline _gsocblogs/2022/blog_ROOT_Neelshah.md:254]:
```suggestion
7) [Final Project Presentation](https://docs.google.com/presentation/d/1VdOEkNsFcaBAo15kjVDRtJ_BWvVh9E6KgKzzrmkr1NQ/edit?usp=sharing)

```

File: _gsocblogs/2022/blog_ROOT_Neelshah.md [hunk 8]
```diff
@@ -158,13 +162,97 @@ CASE-2 \: We look then to the shape values, and if they are equal result is same
...
+8) [GSOC Project Page](https://summerofcode.withgoogle.com/programs/2022/projects/vEuHzl6G)
```
Comment (agheata) [inline _gsocblogs/2022/blog_ROOT_Neelshah.md:255]:
```suggestion
8) [GSOC Project Page](https://summerofcode.withgoogle.com/programs/2022/projects/vEuHzl6G)

```

File: _gsocblogs/2022/blog_ROOT_Neelshah.md [hunk 9]
```diff
@@ -158,13 +162,97 @@ CASE-2 \: We look then to the shape values, and if they are equal result is same
...
 
 ## Proposal Link \: [Inference Code Generation for Deep Learning models](https://docs.google.com/document/d/1tvTY9Rq5j0XW8KFfvkWHWlmE__-94h-Nj0tCkMjYnzs/edit?usp=sharing)
```
Comment (agheata) [inline _gsocblogs/2022/blog_ROOT_Neelshah.md:257]:
```suggestion
## Proposal Link \: 
[Inference Code Generation for Deep Learning models](https://docs.google.com/document/d/1tvTY9Rq5j0XW8KFfvkWHWlmE__-94h-Nj0tCkMjYnzs/edit?usp=sharing)
```

**Review by agheata — CHANGES_REQUESTED** [review]
Formatting changes that I will apply

**Review by agheata — APPROVED** [review]

**Pull request merged**